pkg/domain/infra/tunnel/runtime.go

```diff
@@ -18,3 +24,25 @@ type ContainerEngine struct {
 type SystemEngine struct {
 	ClientCtx context.Context
 }
+
+// RemoteProxySignals enables sig-proxy in remote mode.
+func RemoteProxySignals(ctrID string, killFunc func(string) error) {
```

Comment:
I think this can be a private function as it's exclusively used in the `tunnel` package.

Reply:
done